src/loaders/jobs/docker_img/Dockerfile.microtrait

```diff
@@ -0,0 +1,36 @@
+FROM mambaorg/micromamba:1.4.1
+
+# Note from micromamba - using an environment name other than "base" is not recommended
+ENV CONDA_ENV base
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER /src/loaders/jobs/docker_img/env_microtrait.yaml /tmp/env_microtrait.yaml
+RUN micromamba install -y -n $CONDA_ENV -f /tmp/env_microtrait.yaml && \
+    micromamba clean --all --yes
+
+USER root
+ENV PATH=$PATH:/opt/conda/bin/
+
+# Install microTrait
+RUN Rscript -e 'install.packages("kmed", repos = "https://cloud.r-project.org/")' && \
+    Rscript -e 'devtools::install_github("jlw-ecoevo/gRodon")' && \
+    Rscript -e 'devtools::install_github("ukaraoz/microtrait")'
+
+# Setup of HMM database
+RUN Rscript -e 'library(microtrait)' && \
+    Rscript -e 'microtrait::prep.hmmmodels()' && \
+    Rscript -e 'list.files(file.path(.libPaths(), "microtrait/extdata/hmm/hmmpress"))'
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+RUN echo "source activate $CONDA_ENV" >> ~/.bashrc
```

Comment:
Why does this need to go in `.bashrc` rather than `RUN`ing it?

Reply:
👍 removed. not necessary. especially we are using `base`.

Reply:
:+1: